src/styles/ingredients.scss

```diff
@@ -0,0 +1,182 @@
+.ingredients {
+  margin-bottom: 100px;
+
+  @include tablet {
+    position: relative;
+    margin-bottom: 130px;
+  }
+
+  @include desktop {
+    max-width: 1280px;
+    margin-bottom: 201px;
+  }
+
+  @include big-desktop {
+    max-width: 1440px;
+  }
+
+  &__title {
+    position: relative;
+    max-width: 280px;
+    margin-bottom: 20px;
+
+    @include tablet {
+      margin-bottom: 310px;
+    }
+
+    @include desktop {
+      max-width: 380px;
+      margin-bottom: 287px;
+    }
+  }
+
+  &__blok {
```

Comment:
There appears to be a typographical error in the class name. It is 'ingredients__blok' but you probably meant 'ingredients__block'. Please rename it accordingly.